go.mod

```diff
@@ -17,6 +18,10 @@ require (
 	k8s.io/kube-aggregator v0.22.1
 	k8s.io/utils v0.0.0-20210802155522-efc7438f0176 // indirect
 	open-cluster-management.io/api v0.0.0-20210610125115-f57c747b84aa
+	sigs.k8s.io/apiserver-network-proxy v0.0.24
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.24
 	sigs.k8s.io/apiserver-runtime v1.0.3-0.20210913073608-0663f60bfee2
 	sigs.k8s.io/controller-runtime v0.9.5
 )
+
+replace sigs.k8s.io/apiserver-network-proxy/konnectivity-client => sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.24
```

Comment:
seems useless?

Reply:
oddly yes, otherwise the compiling will fail. i think something's wrong w/ the project's go mod management.